src/activity-log/index.html

```diff
@@ -52,9 +52,9 @@
                     </td>
                     <td class="u-w65pct">
                         <a href="{{ item.permalink }}">
-                            <h4 class="u-mb0">
+                            <span class="h4 u-mb0">
```

Comment:
These shouldn't because they're going to loop through and make them all `th`, but we should have `th` cells in there. I'll open an issue to get back to it when Retirement finishes their work on tables, because it isn't as simple as just adding a `.u-visually hidden` on those cells unfortunately.